content/articles/using-django-in-kivy-application/index.md

```diff
@@ -0,0 +1,596 @@
+### Using Django in Kivy applications.
+
+### Introduction.
+
+Kivy is a popular library for developing applications that can run across all major platforms.
+There may, however, be situations that require your application to access and manipulate data being provided by a web server 
+hosted at a site.In this tutorial, we are going to learn how to make requests to a server
+using the `requests` library. We are going to use Django to develop the server that holds the data we want to manipulate.
```

Comment:
Here, every sentence is in a new line.

If possible, make them into a paragraph. One para must talk about one idea